### ISSUE TITLE
Improve perspective view

### ISSUE THREAD

**Issue by pastk** [organicmaps/organicmaps#2055]
Perspective is useful for navigation in higher speed - it's a way to see further down the road while there is no need to look far to the left or right.
The higher the speed the longer the distance that should be displayed ahead - it could be done by zooming out also.

At the moment in navigation mode the perspective seems to be tied to the current zoom level - more zoom leads to deeper perspective.
And zoom level is tied to the current speed (if auto-zoom is one).
That means that the perspective is at max when the speed is lowest and at min when the speed is the highest, which is wrong. E.g. for pedestrian navigation perspective view is not needed at all I think.

I think it'll be better to tie perspective level to the current speed instead of zoom level: 
* no perspective for low speeds up to e.g. 10kph
* after 10kph the perspective turns on either at a fixed level, or slightly increasing with the speed until a certain limit is reached (e.g. the viewpoint should stay "above" the buildings always)

Then e.g. pedestrians will be able to pan & zoom the map without breaking in and out of perspective view.

**Comment by arnaudvergnet** [comment]
Maybe have the perspective be only applied if the camera is set to follow the user's position? This way it will make panning the map easier when trying to add a stop while in a car, or trying to see ahead (useful for co-pilots).

**Comment by arnaudvergnet** [comment]
It could also be helpful to let the user manually control the perspective with a two finger swipe up or down (in both normal map and navigation mode). This is pretty common behavior in other map applications.

**Comment by Echolon** [comment]
Still, I think there should be some limits before the app tries to be "clever. Sometimes its really annoying that Organic maps zooms out to far while driving. This can be dangerous, as it distracts and potentially forces the driver to adjust the zoom level again while driving.

I would at least ask the user to give a reference or preferred default zoom level.

**Comment by ghost** [comment]
> Still, I think there should be some limits before the app tries to be "clever. Sometimes its really annoying that Organic maps zooms out to far while driving. This can be dangerous, as it distracts and potentially forces the driver to adjust the zoom level again while driving.
> 
> I would at least ask the user to give a reference or preferred default zoom level.

Yeah. Everyone likes different. I prefer overview when there is no maneuver. But look ahead 1,5km at Autobahn at speed from 100kph is way too close.
